virttest/shared/cfg/base.cfg

```diff
@@ -255,6 +255,10 @@ mem = 1024
 # Pattern for check memory size inside guest
 mem_chk_re_str = ([0-9]+)
 
+# this parameter controls the value of "mem-path" in memory-backend object
+# it will be replaced by "hugepage_path"
```

Comment:
What does the mean of the `it will be replaced by "hugepage_path"`?

Reply:
like what we discussed before, the old param "hugepage_path" would overwrite this new one

Reply:
updated